### PR TITLE
Enable passing in a custom pauseAfterLoad option.

### PR DESCRIPTION
**Pull request by samccone**
Also do a tiny bit of option default refactoring.

### CODE REVIEW

File: lighthouse-core/gather/drivers/driver.js [hunk 1]
```diff
@@ -262,7 +263,7 @@ class Driver {
               clearTimeout(loadTimeout);
             }
             resolve();
-          }, PAUSE_AFTER_LOAD);
+          }, _options.pauseAfterLoad || _PAUSE_AFTER_LOAD);
```
Comment (paulirish) [inline lighthouse-core/gather/drivers/driver.js]: update the const at the top?

Comment (samccone) [inline lighthouse-core/gather/drivers/driver.js]: fixed

**Comment by brendankenny** [comment]
Are you using driver directly? Should this be normalized in runner.js?

**Comment by samccone** [comment]
no @brendankenny 
image

**Pull request merged**